internal/gitlab/gitlab.go

```diff
@@ -440,14 +443,17 @@ func ChangeRegistryToGitLab(dryRun bool) {
 			url: {{ .FullURL }}
 		kind: Secret
 		metadata:
-		annotations:
-			managed-by: argocd.argoproj.io
-		labels:
-			argocd.argoproj.io/secret-type: repository
-		name: repo-gitlab
-		namespace: argocd
+			annotations:
+				managed-by: argocd.argoproj.io
+			labels:
+				argocd.argoproj.io/secret-type: repository
+			name: repo-gitlab
+			namespace: argocd
 		type: Opaque
-	`)
+		`)
+		if err != nil {
```

Comment:
in case `err is nil` , next condition will never happen

Reply:
I believe thata is the idea, no?
- If the template don't work, we can execute it application.. 
- Am I handling the same error twice? is this your concern?